openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -69,7 +64,7 @@ def render_to_fragment(self, request, course_id, user_access, **kwargs):
         course_home_messages = list(CourseHomeMessages.user_messages(request))
 
         # Pass in the url used to set a course goal
-        goal_api_url = reverse('course_goals_api:v0:course_goal-list', request=request)
+        goal_api_url = get_goals_api_url(request)
```

Comment:
Nit: If you like "goals_api_url" (as in "get_goals_api_url"), try to be consistent.  I know I pushed toward "goal_api" in some places due to that doc.  Feel free to ignore those comments from last PR.